lib/Command/Updater/UpdateUser.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Nextcloud - News
+ *
+ * This file is licensed under the Affero General Public License version 3 or
+ * later. See the COPYING file.
+ */
+
+namespace OCA\News\Command\Updater;
+
+use Exception;
+use OCA\News\Service\FeedServiceV2;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class UpdateUser extends Command
+{
+    /**
+     * @var FeedServiceV2 Feed service
+     */
+    private $feedService;
+
+    public function __construct(FeedServiceV2 $feedService)
+    {
+        parent::__construct();
+        $this->feedService = $feedService;
+    }
+
+    /**
+     * @return void
+     */
+    protected function configure()
+    {
+        $this->setName('news:updater:update-user')
+            ->addArgument(
+                'user-id',
+                InputArgument::REQUIRED,
+                'user id of a user, string'
+            )
+            ->setDescription('Console API for updating a single user\'s feed');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $userId = $input->getArgument('user-id');
+        $feeds = $this->feedService->findAllForUser($userId);
+        $updateErrors = false;
+        foreach ($feeds as $feed) {
```

Comment:
Why not call the updater service for this? To align it better with the cronjob

Reply:
I don't fully understand your suggestion. The `UpdaterService` has an `update` method, which looks like it updates all feeds for all users. Are you suggesting that I add a new method in `UpdaterService` called `updateForUser`, call that, and then move the logic there?

Reply:
I thought it was user based. In that case I'll leave it up to you, either option is acceptable to me.